frontend/pages/Parts/index.tsx

```diff
@@ -33,7 +33,7 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
          getGlobalSettings(),
          getStoreList(),
          getStoreByCode('us'),
-         findProductList({ handle: { eq: 'Parts' } }),
+         findProductList({ handle: { eqi: 'Parts' } }),
```

Comment:
Not sure it matters but I think this one didn't have to change.

Reply:
https://github.com/iFixit/react-commerce/pull/556/commits/0657ba2256fd53073aa3f547ba4dd6a4aa597384